### ISSUE TITLE
Incompatible Node Version

### ISSUE THREAD

**Issue by djthornton1212** [NickLiffen/ghas-enablement#53]
# Problem:
Yarn states that this module is incompatible with my version of node. The documentation states 
> Node v16 or higher installed

I'm running 17 and it fails. I can see that in the package.json it explicitly sets:

```
"engines": {
    "node": "16"
}
```

# System Info:

System OS: Windows
NodeJS: 17.5.0

# Logs:

> yarn run getOrgs     
> yarn run v1.22.15
> error github-helper@1.0.0: The engine "node" is incompatible with this module. Expected version "16". Got "17.5.0"
> error Commands cannot run with an incompatible environment.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> 
> yarn -v
> 1.22.15
> 
> node -v
> v17.5.0

# Expected behavior:
The module should accept engines versioned 16 or higher.

# Suggested fix:
```
"engines": {
    "node": ">=16"
}
```

**Comment by NickLiffen** [comment]
Fixed #55 👍 

Thanks for the feedback :)

**Comment by djthornton1212** [comment]
Happy to help. Thanks for creating this life/timesaver.
